### PR TITLE
Iss 139 homepage redesign

### LINKED ISSUE #2: Homepage redesign - very important task
### **This will be the redesign:  [Google Slides - Homepage redesign](https://docs.google.com/presentation/d/1Cov51q6ufnUk30gwp5jmm-Ckq6wpO5GOpSNS0jPp0Zk/edit#slide=id.p).**

This is a very important task and the faster it gets done the better. Our current www.frempco.com homepage is bare. Every time I speak to anyone about Frempco, I need to explain from scratch what Frempco is about.  It would be much much faster to get teachers to try out Frempco if the website was self-explanatory. The faster this redesign gets landed in production, the faster I can reach out to teachers and give them free trials of Frempco. 

Below are the screenshots for the new homepage
Student page Lord of the Flies
Teacher page Lord of the Flies

### PR DESCRIPTION
**Pull request by karlrez**
Resolves #2  and reopening PR [147](https://github.com/mssiegel/frempco-client/pull/147) from old repo

### CODE REVIEW

File: src/components/pages/TeachersPage/index.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
   }
 
   return (
-    <main>
+    <main style={{padding: '16px'}}>
```
Comment (mssiegel) [inline src/components/pages/TeachersPage/index.tsx:134]:
```suggestion
    <main>
```

File: src/components/pages/StudentsPage/index.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ export default function StudentsPage({ classroomName }: ClassroomProps) {
   }, [router.events, socket]);
 
   return (
-    <main>
+    <main style={{padding: '16px'}}>
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/index.tsx:66]:
```suggestion
    <main>
```

**Review by mssiegel — APPROVED** [review]
Thanks @karlrez.  This is almost ready. A few minor points:
- I think the TeacherOverview image is accidentally saved twice in public folder
- i left comments on two places where i think 16px padding was added in accidentally.

Now I understand what you mean by there's a horizontal scroll on small mobile devices. I see it when using mobile device with dev tools. I looked up on stack overflow and it says that using overflowX should be avoided. overflowX treats the symptoms. But its better to solve the root cause by fixing the css causing the problem.  See: https://stackoverflow.com/a/15089567   

Once addressed, go ahead and merge. Whoohoo. Thanks Karl!

Karl, after merging this, can you make a new GitHub issue to remind us to eventually address the horizontal mobile scroll? You can give it the label of "low-priority"

File: src/components/pages/TeachersPage/index.tsx (hunk 1 above)
Comment (karlrez) [reply src/components/pages/TeachersPage/index.tsx:134]: For the padding here and in StudentsPage/index.tsx I don't see this on my local. It was added in the first commit and then removed in the second commit. If you checkout the branch on your local does it still show?

**Comment by karlrez** [comment]
hey @mssiegel regarding the horizontal scroll, the issue is from the second List, the list items are overflowing. Alternatively, we can also:
- set a smaller width to the list items (98% worked fine)
- reduce marginLeft on the list items

But yea I am also fine to leave for later as a seperate issue.

**Comment by mssiegel** [comment]
Yeah, when I pull it locally the TeachersOverview duplicate image and the extra padding show in my local. Are you in the branch for this new repo or for the old repo?   Strange you don't have them in your local. I'd say go ahead and merge this PR as is. After the merge we'll make a new PR to delete them. If for some reason you still don't see it in your local then I'll make the PR to delete them. 

Regarding the horizontal scroll, lets leave it for later as a separate issue. This way nothing will block this PR from getting merged. You can put your proposed solution as a comment in the issue.

**Comment by karlrez** [comment]
ah after I closed everything and reopened again I caught it, must have been a mix up earlier. I'll merge and create an issue for the horizontal scroll.

**Pull request merged**